lib/morse_translator.rb

```diff
@@ -38,6 +40,75 @@ def initialize
                     "9" => "----.",
                     "0" => "-----",
                     " " => " "}
+
+
   end
 
+  def translate(string)
+    string_array = []
+    holder = string.downcase.chars
+    holder.each do |char|
+      dictionary.each do |key,val|
```

Comment:
I LOVE that you are iterating over the dictionary hash here!  So cool!

There is a slightly less complicated way to make the translation, see below, but what you have here is GREAT!

```ruby
def translate(string)
  string_array = []
  holder = string.downcase.chars
  holder.each do |char|
    string_array << @dictionary[char]
  end
  string_array.join("")
end
```

Maybe review the hash lesson and practice accessing values in a hash using just the key!